### PR TITLE
Update Func_Graphs.mdx

### PR DESCRIPTION
**Pull request by abhideepd**
adding the additional solution, below solution 2

Previous pr reference: https://github.com/cpinitiative/usaco-guide/pull/4331#issuecomment-1953027997

_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [x] I have linked this PR to any issues that it closes. https://github.com/cpinitiative/usaco-guide/issues/4330

### CODE REVIEW

**Review by danielzsh — DISMISSED** [review]

**Comment by SansPapyrus683** [comment]
you don't have to make a new pr each time, just commit and push your changes to the og pr

File: content/3_Silver/Func_Graphs.mdx [hunk 1]
```diff
@@ -716,6 +716,75 @@ print(" ".join(str(i + 1) for i in ans))
...
+<LanguageSection>
+<JavaSection>
+
+```java
+import java.io.*;
+import java.util.*;
+
+public class Codeforces_Badge {
+	static BufferedReader x =
+	    new BufferedReader(new InputStreamReader(System.in));
```
Comment (SansPapyrus683) [inline content/3_Silver/Func_Graphs.mdx:733]: move this in `main`

File: content/3_Silver/Func_Graphs.mdx [hunk 2]
```diff
@@ -716,6 +716,75 @@ print(" ".join(str(i + 1) for i in ans))
...
+		int n = Integer.parseInt(st.nextToken());
+		int arr[] = new int[n + 1];
+		st = new StringTokenizer(x.readLine());
+		for (int i = 1; i <= n; i++) {
+			arr[i] = Integer.parseInt(st.nextToken());
+		}
+		getAnswer(arr, n);
+	}
+
+	static void getAnswer(int arr[], int n) {
```
Comment (SansPapyrus683) [inline content/3_Silver/Func_Graphs.mdx:746]: this too

File: content/3_Silver/Func_Graphs.mdx [hunk 3]
```diff
@@ -716,6 +716,75 @@ print(" ".join(str(i + 1) for i in ans))
 </PySection>
 </LanguageSection>
 
+### Solution 3
+
+It's also possible to use `floyd(x)` to generate answers for all vertices in the connected component
+containing `x` without using adjacency lists.
```
Comment (SansPapyrus683) [inline content/3_Silver/Func_Graphs.mdx:722]: what do you use in place of adj lists?

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

**Comment by danielzsh** [comment]
By the way, @abhideepd make sure to copy the latest changes on this branch into the editor before making further edits so you don't accidentally override our edits; the editor doesn't auto-pull yet 😓

**Comment by abhideepd** [comment]
> you don't have to make a new pr each time, just commit and push your changes to the og pr

sure will do, hence forth.
pardon me, new to git, still a noob 😅

**Comment by abhideepd** [comment]
> By the way, @abhideepd make sure to copy the latest changes on this branch into the editor before making further edits so you don't accidentally override our edits; the editor doesn't auto-pull yet 😓

sure will do it, still a noob in bit 😅. 
I wasn't able to use the editor though, edited directly on github itself.

**Comment by abhideepd** [comment]
shit, i closed it again, by mistake 😓 my bad, while synching my fork, i removed all my commits. sorry, a new pr is coming ahead

**Pull request closed without merging**